tests/src/system/basic/WskBasicTests.scala

```diff
@@ -688,9 +688,9 @@ class WskBasicTests
 
     behavior of "Wsk Namespace CLI"
 
-    it should "list namespaces" in {
+    it should "return a list of exactly one namespace" in {
         wsk.namespace.list().
-            stdout should include regex ("@|guest")
+            stdout.lines should have size 2 // headline + namespace
```

Comment:
No, `wskadmin` is not to be used in a system level test. Pressure on the `whoami` endpoint rises.